config/config.txt

```diff
@@ -195,6 +195,14 @@ CHECK_RANDOMIZER
 ## Ban appeals URL - usually for a forum or wherever people should go to contact your admins.
 # BANAPPEALS http://justanotherday.example.com
 
+## System command that invokes youtube-dl, used by Play Internet Sound.
+## You can install youtube-dl with
+## "pip install youtube-dl" if you have pip installed
+## from https://github.com/rg3/youtube-dl/releases
+## or your package manager
+## The default value assumes youtube-dl is in your system PATH
```

Comment:
For a moment I thought I won the "longest config option description" award, but USE_AGE_RESTRICTION_FOR_JOBS still claims that prize.